### PR TITLE
doc: add flutter setup instructions

### PR DESCRIPTION
**Pull request by vivianjeng**
- init documentation for flutter setup
- example: https://github.com/zkmopro/flutter-app
- ~add instructions how to add the plugin~

### CODE REVIEW

File: docs/docs/setup/flutter-setup.md [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+```
+flutter-app/
+├── ...
+├── assets/multiplier2_final.zkey
+└── lib/main.dart
+```
+
+1. Place your `.zkey` file in your app's assets folder and remove the example file `assets/multiplier2_final.zkey`. If your `.zkey` has a different file name, don't forget to update the asset definition in your app's `pubspec.yaml`:
+
+   ```diff
```
Comment (vivianjeng) [inline docs/docs/setup/flutter-setup.md:133]: `diff` is not highlighted in docusaurus by default
you should add the supported languages (`diff`) here

https://github.com/zkmopro/mopro/blob/530a2df8bc64f1463a30d1c685f0bfcac62f498c/docs/docusaurus.config.ts#L44

**Review by vivianjeng — COMMENTED** [review]
I think you can also add step numbers in each step
e.g. 
1. Running the example app
2. Integrate your ZKP

...
so users will better understand which step they are and we can communicate easily (e.g. step 2 doesn't work)

(but I haven't done that with iOS and android setups)

And I think we can also suppose that if developers already have their flutter apps
and how they can integrate with the flutter plugin
like this section [1. Initializing a New React Native Project or Using an Existing One](https://zkmopro.org/docs/setup/react-native-setup#1-initializing-a-new-react-native-project-or-using-an-existing-one)
but this can be done in the future PR
the current PR is good and complete enough for getting started with flutter

File: docs/docs/setup/flutter-setup.md [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
+   flutter run
+   ```
+
+## Integrating Your ZKP
+
+The example app comes with a simple prover generated from a Circom circuit. To integrate your own prover, follow the steps below.
+
+### Setup
+
+Follow the [Rust Setup steps from the mopro official docs](https://zkmopro.org/docs/getting-started/rust-setup) to generate the platform-specific libraries.
```
[the remark this answers is not in the document]
Comment (alxkzmn) [reply docs/docs/setup/flutter-setup.md:80]: I'll fix the link, but do you also mean that this section is deprecated? https://zkmopro.org/docs/setup/rust-setup

File: docs/docs/setup/flutter-setup.md [hunk 3]
```diff
@@ -0,0 +1,178 @@
...
+
+2. For iOS:
+
+   Make sure you have an iOS simulator running or a device connected. Then run:
+
+   ```bash
+   flutter run
+   ```
+
+## Integrating Your ZKP
```
[the remark this answers is not in the document]
Comment (vivianjeng) [reply docs/docs/setup/flutter-setup.md:74]: How about this @alxkzmn ?

[the remark this answers is not in the document]
Comment (alxkzmn) [reply docs/docs/setup/flutter-setup.md:74]: I'm still working on the PR, will let you know when it's ready for review :)

File: docs/versioned_docs/version-0.1.0/setup/flutter-setup.md [hunk 4]
```diff
@@ -0,0 +1,191 @@
...
+   + proofResult = await _moproFlutterPlugin.generateProof("assets/your_new_zkey_file.zkey", inputs);
+   ```
+
+Don't forget to modify the input values for your specific case!
+
+## 2.2 Integrating mopro Into Existing Flutter App
+
+If you already have a Flutter project, follow the steps below to integrate mopro.
+
+1. Copy the `mopro_flutter_plugin` directory from the repository root into the root folder of your existing Flutter project:
```
Comment (vivianjeng) [inline docs/versioned_docs/version-0.1.0/setup/flutter-setup.md:143]:
```suggestion
1. Copy the `mopro_flutter_plugin` directory from [the repository](https://github.com/zkmopro/flutter-app) root into the root folder of your existing Flutter project:
```

**Review by vivianjeng — COMMENTED** [review]
LGTM
Thank you!

**Pull request merged**